### PR TITLE
Re-Names Most Cyberiad References

### PR DESCRIPTION
**Pull request by Cid777**
Cyberiad - Stratus
NSS Cyberiad - NTDSF Stratus
N.S.S. Cyberiad - N.T.D.S.F. Stratus
NanoTrasen Science Station Cyberiad - NanoTrasen Deep Space Facility Stratus

Apologies if there's any issues, my first time using the Github.

### CODE REVIEW

File: code/modules/awaymissions/maploader/reader.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@ var/global/dmm_suite/preloader/_preloader = new
  *
  * If you need to freeze init while you're working, you can use the spacial allocator's
  * "add_dirt" and "remove_dirt" which will put initializations on hold until you say
- * the word. This is important for loading large maps such as the cyberiad, where
+ * the word. This is important for loading large maps such as the stratus, where
```
Comment (Codingale) [inline code/modules/awaymissions/maploader/reader.dm]: Comments aren't seen by most, so probably a bit silly to change everything but meh.

**Comment by Codingale** [comment]
Said it's most, are there still some?

File: code/modules/awaymissions/maploader/reader.dm (hunk 1 above)
Comment (Cid777) [inline code/modules/awaymissions/maploader/reader.dm]: You're right, it is silly. I just changed it so I could keep using Ctrl + F to find more references.

**Comment by Cid777** [comment]
I didn't touch any that actually involved file names in any way. Those are still called Cyberiad.

**Pull request merged**